### PR TITLE
Update pre-commit black and isort version

### PR DESCRIPTION
**Pull request by hashlash**
This would solve the failed github action

See:
- https://github.com/psf/black/issues/2964
- https://github.com/PyCQA/isort/issues/2077

### CODE REVIEW

**Comment by hashlash** [comment]
It seems that  #167 needs to be merged first.

**Comment by dfunckt** [comment]
It’s failing on what seems like installation failure for isort, maybe include that commit in your PR to run them both in one go here?

Python reminds me of JavaScript a lot lately…

**Comment by hashlash** [comment]
Updated @dfunckt :rocket:

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -10,11 +10,11 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
           fetch-depth: 0
       - name: Set up Python 3.10
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v3
```
Comment (cclauss) [inline .github/workflows/ci.yml:17]:
```suggestion
        uses: actions/setup-python@v4
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -38,14 +38,14 @@ jobs:
           - python-version: "3.x"
             toxenv: "djangomain"
             experimental: true
-          - python-version: "pypy3"
+          - python-version: "pypy-3.8"
```
Comment (cclauss) [inline .github/workflows/ci.yml:41]: Why not 3.9?

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -38,14 +38,14 @@ jobs:
...
             toxenv: "PyPy"
             experimental: false
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
 
-    - uses: actions/setup-python@v2
+    - uses: actions/setup-python@v3
```
Comment (cclauss) [inline .github/workflows/ci.yml:48]:
```suggestion
    - uses: actions/setup-python@v4
```

**Comment by dfunckt** [comment]
OK, checks pass -- I'll merge this one to unblock CI and we can bump pypy and action versions separately.

**Comment by dfunckt** [comment]
Thanks @hashlash and @cclauss !

**Pull request merged**